src/history/file_backed.rs

```diff
@@ -77,7 +77,12 @@ impl History for FileBackedHistory {
     fn load(&self, id: HistoryItemId) -> Result<super::HistoryItem> {
         Ok(FileBackedHistory::construct_entry(
             Some(id),
-            self.entries[id.0 as usize].clone(),
+            self.entries
+                .get(id.0 as usize)
+                .ok_or(ReedlineError(ReedlineErrorVariants::OtherHistoryError(
+                    "Item does not exist",
+                )))?
+                .clone(),
```

Comment:
Did you run into problems here that caused a panic? Should we fix that if necessary?

Reply:
When testing, I had a bug that sent the i64::MAX-1 used for the filtered-item id to the wrapped history, which caused a panic. But I don't think any actual code ever uses an ID that doesn't exist.